### ISSUE TITLE
how can I debug the reloading code step by step in IDE such as vscode?

### ISSUE THREAD

**Issue by yanzhen74** [julvo/reloading#16]
Once step over reloading part, one step will exec to end.

**Comment by agrigori-qc** [comment]
Agree, would be fantastic with debugging capabilities for the reloaded code.
With vscode I cannot step into the function and it does not break on breakpoints

**Comment by neoxharsh** [comment]
In order to debug while using reload, make modification as below
1. In get_loop_code
return compile(tree, filename=fpath, mode="exec"), format_itervars(itervars), found_loop_id
2. In get_function_def_code
compiled = compile(tree, filename=fpath, mode="exec")
